sdk/eventhubs/azure_messaging_eventhubs/examples/eventhubs_tracing_subscriber.rs

```diff
@@ -121,6 +131,42 @@ where
             eprintln!("Failed to send event to EventHubs");
         });
     }
+
+    fn on_new_span(&self, attrs: &span::Attributes<'_>, id: &span::Id, _ctx: Context<'_, S>) {
+        let event_data = EventData::builder()
+            .add_property(EVENT_TYPE_PROPERTY.into(), SPAN_OPEN)
+            .add_property(LOG_LEVEL_PROPERTY.into(), attrs.metadata().level().as_str())
+            .add_property(SPAN_ID_PROPERTY.into(), id.into_u64())
+            .add_property(
+                TIMESTAMP_PROPERTY.to_string(),
+                SystemTime::now()
+                    .duration_since(SystemTime::UNIX_EPOCH)
+                    .unwrap()
+                    .as_secs(),
+            )
+            .add_property(NAME_PROPERTY.to_string(), attrs.metadata().name())
+            .build();
+        self.sender.try_send(event_data).unwrap_or_else(|_| {
+            eprintln!("Failed to send span open event to EventHubs");
+        });
+    }
+    fn on_close(&self, id: span::Id, _ctx: Context<'_, S>) {
+        // This method is called when a span is closed.
+        let event_data = EventData::builder()
+            .add_property(EVENT_TYPE_PROPERTY.into(), SPAN_CLOSE)
+            .add_property(NAME_PROPERTY.into(), id.into_u64())
```

Comment:
Shouldn't this be `SPAN_ID_PROPERTY` to match with line 139 above? You use `NAME_PROPERTY` for `attrs.metadata().name()`.

Reply:
Good catch, it absolutely is a typo.

I have a stupid number of unrelated changes to AMQP (cleaning up a TON of errant `.clone()` calls). I'll get this and one other fix in with my next PR (which will be adding a "read events from EventHub" sample which is the inverse of this sample).